x/incentive/keeper/rewards.go

```diff
@@ -373,8 +382,8 @@ func (k Keeper) SynchronizeHardBorrowReward(ctx sdk.Context, borrow hardtypes.Bo
 			continue
 		}
 
-		userRewardIndexes, foundUserRewardIndexes := claim.BorrowRewardIndexes.GetRewardIndex(coin.Denom)
-		if !foundUserRewardIndexes {
+		userMultiRewardIndex, foundUserMultiRewardIndex := claim.SupplyRewardIndexes.GetRewardIndex(coin.Denom)
```

Comment:
```suggestion
		userMultiRewardIndex, foundUserMultiRewardIndex := claim.BorrowRewardIndexes.GetRewardIndex(coin.Denom)
```

Reply:
Fixed in https://github.com/Kava-Labs/kava/pull/833/commits/8d16c3e8c3f3efdd601687310fa9e952306921ac